opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/FilterQueryBuilderTest.java

```diff
@@ -634,6 +636,93 @@ void should_build_match_phrase_query_with_custom_parameters() {
                 dsl.namedArgument("zero_terms_query", literal("ALL")))));
   }
 
+  @Test
+  void wildcard_query_invalid_parameter() {
```

Comment:
Needs test that show how escape characters -- `\%` and `\_` -- are handled.

Reply:
Added in 0b9752ca994a334de8e9bbc6bd7ac49fadd37944